languages/cpp/exercises/concept/enums/strings_test.cpp

```diff
@@ -0,0 +1,87 @@
+#include "strings.h"
+#include "test/catch.hpp"
+
+using namespace std;
+
+TEST_CASE("Error message")
```

Comment:
I'm slightly confused about the contents of this PR. It looks like this test file still contains the contents of the `strings` exercise? Was that intentional?